full_bubble_model.py

```diff
@@ -634,80 +669,82 @@ def solve(cpar, t_int=np.array([0.0, 1.0]), LSODA_timeout=30.0, Radau_timeout=30
         
     error_code = 0
     start = time.time()
-    num_sol = None
     if not check_cpar(cpar):
         error_code += 300
+        if print_errors:
+            print(colored(error_codes['3xx']['describtion'], error_codes['3xx']['color']))
         return None, error_code, 0.0
     
+    # Arguments and IC for _f()
     ex_args = []
     for key in excitation_args:
         ex_args.append(cpar.get(key, 0.0))
     ex_args = np.array(ex_args, dtype=np.float64)
+    args=(cpar.P_amb, cpar.alfa_M, cpar.T_inf, cpar.surfactant, cpar.P_v, cpar.mu_L, cpar.rho_L, cpar.c_L, ex_args, extra_dims) # _f()'s arguments
     IC, lowpressure_error, lowpressure_warning = _initial_condition(cpar, enable_evaporation, extra_dims)
     if lowpressure_error:
         error_code += 100
+        if print_errors:
+            print(colored(error_codes['1xx']['describtion'], error_codes['1xx']['color']))
         return None, error_code, 0.0
     elif lowpressure_warning:
         error_code += 200
+        if print_errors:
+            print(colored(error_codes['2xx']['describtion'], error_codes['2xx']['color']))
     
     # solving d/dt x=f(t, x, cpar)
-    try: # try-catch block
-        num_sol = func_timeout( # timeout block
-            LSODA_timeout, solve_ivp,
-            kwargs=dict(fun=_f, t_span=t_int, y0=IC, method='LSODA', atol = 1e-10, rtol=1e-10, # solve_ivp()'s arguments
-                        args=(cpar.P_amb, cpar.alfa_M, cpar.T_inf, cpar.surfactant, cpar.P_v, cpar.mu_L, cpar.rho_L, cpar.c_L, ex_args, extra_dims) # _f()'s arguments
-            )
-        )
-        if num_sol.success == False:
+    # Try with LSODA
+    num_sol1 = scipy_ivp.solve_ivp(fun=_f, t_span=t_int, y0=IC, method='LSODA', timeout=LSODA_timeout, args=args,
```

Comment:
`de.solve()` is completely reworked, no more try-except blocks and no func_timeout. `num_sol `is always provided, except for invalid control parameters. Errors with solver messages are printed in flight. See` scipy_ivp.py` for argument compression.